helm-swoop.el

```diff
@@ -542,7 +534,7 @@ If $linum is number, lines are separated by $linum"
          (mapconcat 'identity
                     (split-string helm-pattern " ")
                     "\\|")))
-    (when (or (and (and (featurep 'migemo) helm-migemo-mode)
+    (when (or (and (featurep 'migemo) helm-migemo-mode
```

Comment:
Put form `helm-migemo-mode` at newline for readability

Reply:
I understand. but `(featurep 'migemo) helm-migemo-mode` is strongly connection.
Or I suggest that you make this and in one line, what do you think?

Reply:
I'm OK with it.